lib/taskana-core/src/main/java/pro/taskana/impl/persistence/MapTypeHandler.java

```diff
@@ -17,16 +17,16 @@
  *
  * @author EH
  */
-@SuppressWarnings("rawtypes")
-public class MapTypeHandler extends BaseTypeHandler<Map> {
+public class MapTypeHandler extends BaseTypeHandler<Map<String, Object>> {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MapTypeHandler.class);
 
   @Override
-  public void setNonNullParameter(PreparedStatement ps, int i, Map parameter, JdbcType jdbcType)
+  public void setNonNullParameter(
+      PreparedStatement ps, int i, Map<String, Object> parameter, JdbcType jdbcType)
       throws SQLException {
     if (parameter != null && parameter.size() > 0) {
-      LOGGER.debug("Input-Map before serializing: ", parameter);
+      LOGGER.debug("Input-Map before serializing: {}", parameter);
```

Comment:
depending on map size ... this could be time consuming. is this a problem?

Reply:
it's debug. Not production relevant. 
+ As soon as we get slf4j 2.0 we will use the lambda method and thus this will be no issue